scripts/a11y/process_a11y.py

```diff
@@ -134,11 +134,13 @@ def get_error_category(self, code):
     @staticmethod
     def read_csv(filename):
         with open(filename, 'r') as f:
-            return [row for row in csv.reader(f)]
+            reader = csv.reader(f)
+            next(reader)  # TODO: make header row skip configurable
+            return [row for row in reader]
 
 
 if __name__ == '__main__':
-    a11y_filename = 'results/a11y.csv'
+    a11y_filename = 'data/a11y.csv'
```

Comment:
shouldn't this be `results`? that's where the CSV is saved to once a scan is done.